src/hotspot/cpu/ppc/macroAssembler_ppc.cpp

```diff
@@ -2644,14 +2196,6 @@ void MacroAssembler::compiler_fast_lock_object(ConditionRegister flag, Register
     bne(flag, failure);
   }
 
-#if INCLUDE_RTM_OPT
-  if (UseRTMForStackLocks && use_rtm) {
```

Comment:
Can RTM related method parameters (e.g. `use_rtm`) be removed?

Reply:
Good catch! There are even more parameters which are no longer needed. And I've found some more leftovers. I just removed all of them.